libcloudforensics/gcp.py

```diff
@@ -296,14 +296,20 @@ def GetDisk(self, disk_name, zone=None):
       zone = disk['zone']
     return GoogleComputeDisk(self, zone, disk_name)
 
-  def CreateDiskFromSnapshot(
-      self, snapshot, disk_name=None, disk_name_prefix=''):
+  def CreateDiskFromSnapshot(self,
+                             snapshot,
+                             disk_name=None,
+                             disk_name_prefix='',
+                             disk_type='pd-standard'):
     """Create a new disk based on a Snapshot.
 
     Args:
       snapshot (GoogleComputeSnapshot): Snapshot to use.
       disk_name (str): Optional. String to use as new disk name.
       disk_name_prefix (str): Optional. String to prefix the disk name with.
+      disk_type (str): Optional. URL of the disk type resource describing
```

Comment:
Add something about running `gcloud compute disk-types list` to get a full list of available types?